configure.ac

```diff
@@ -1316,6 +1316,10 @@ AC_ARG_ENABLE(testbench,
 )
 AM_CONDITIONAL(ENABLE_TESTBENCH, test x$enable_testbench = xyes)
 if test "x$enable_testbench" = "xyes"; then
+	AC_CHECK_PROG(IP, [ip], [yes], [no])
```

Comment:
I think we should make this optional, as only one test needs it. The test should skip out if ip is not found. As it is now, the testbench does not work at all on any platform that does not have "ip".